package.json

```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "autosuggest-highlight": "^3.1.0",
-    "material-ui": "1.0.0-beta.17",
+    "material-ui": "1.0.0-beta.21",
     "prop-types": "^15.6.0",
```

Comment:
FYI, the new recommendation is to use flowtype instead of prop-types

Reply:
Is there a link/resource that states/explains the recommendation? Ill write some issues/tasks to investigate this and move to it eventually.